packages/neuron-wallet/src/controllers/cell-manage.ts

```diff
@@ -0,0 +1,84 @@
+import type { OutPoint, Script } from '@ckb-lumos/base'
+import { outPointTransformer } from '../database/chain/entities/cell-local-info'
+import Output from '../models/chain/output'
+import CellsService, { LockScriptCategory, TypeScriptCategory } from '../services/cells'
+import WalletService from '../services/wallets'
+import CellLocalInfoService from '../services/cell-local-info'
+import { AddressNotFound, CurrentWalletNotSet } from '../exceptions'
+import AddressService from '../services/addresses'
+import { scriptToAddress } from '../utils/scriptAndAddress'
+import SignMessage from '../services/sign-message'
+import NetworksService from '../services/networks'
+
+export default class CellManage {
+  static async getLiveCells() {
+    const currentWallet = WalletService.getInstance().getCurrent()
+    if (!currentWallet) throw new CurrentWalletNotSet()
+    const liveCells = (await CellsService.getLiveOrSentCellByWalletId(currentWallet.id, { includeCheque: true })).map(
+      v => v.toModel()
+    )
+    const outPoints = liveCells.filter((v): v is Output & { outPoint: OutPoint } => !!v.outPoint).map(v => v.outPoint)
+    const cellLocalInfoMap = await CellLocalInfoService.getCellLocalInfoMap(outPoints)
+    return liveCells
+      .map<
+        Pick<Output, 'capacity' | 'outPoint' | 'lock' | 'type' | 'timestamp'> & {
+          lockScriptType: LockScriptCategory
+          typeScriptType?: TypeScriptCategory
+          description?: string
+          locked?: boolean
+        }
+      >(v => {
+        const result = {
+          capacity: v.capacity,
+          outPoint: v.outPoint,
+          lock: v.lock,
+          type: v.type,
+          data: v.data,
+          timestamp: v.timestamp,
+          lockScriptType: CellsService.getCellLockType(v),
+          typeScriptType: CellsService.getCellTypeType(v),
+        }
+        if (!v.outPoint) {
+          return result
+        }
+        const outPointKey = outPointTransformer.to(v.outPoint)
+        return {
+          ...result,
+          description: cellLocalInfoMap[outPointKey]?.description,
+          locked: cellLocalInfoMap[outPointKey]?.locked,
+        }
+      })
+      .sort((a, b) => +(b.timestamp ?? 0) - +(a.timestamp ?? 0))
+  }
+
+  static updateLiveCellLocalInfo(outPoint: OutPoint, locked?: boolean, description?: string) {
```

Comment:
I would suggest using an object instead of parameter list to make it easy to extend, e.g. `(outPoint: OutPoint, attributes: Partial<{ locked: boolean, description: string }>`.
By doing so, attributes can be extended without updating the interface.